test/r2.md

```diff
@@ -0,0 +1,440 @@
+For example, it is enough to change the last hex-digit from "a" to "b", e.g.
+hash string should be like that
+`ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12b`
+
+c) Run the `set_swupgrade_repo` command with wrong hash string:
+
+```bash
+provisioner set_swupgrade_repo 2.0.1 --source=/root/single_cortx.iso --hash="ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12b"  --hash-type="sha256"
+```
+
+The exception output should be something like that and provisioner should stop command execution:
+```
+'repo source "/root/single_cortx.iso" is not acceptable, reason: Catalog structure validation error occurred:
+"Validation Failed. Reason: Hash sum of file \'/root/single_cortx.iso\': \'ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a\' mismatches the provided one \'ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12b\'"'
+```
+
+d) With correct hash sum `ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a` run the `set_swupgrade_repo` command with wrong `--hash-type` parameter
+
+```bash
+
+provisioner set_swupgrade_repo 2.0.1 --source=/root/single_cortx.iso --hash="ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a"  --hash-type="md5"
+
+```
+
+Output should be the same as in c) sub-step.
+
+e) With correct hash sum `ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a` run the `set_swupgrade_repo` command with missed hash-prefix and without `--hash-type` parameter.
+Be default we use MD5 hash sum
+
+```bash
+
+provisioner set_swupgrade_repo 2.0.1 --source=/root/single_cortx.iso --hash="ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a"
+
+```
+
+Output should be the same as in c) sub-step.
+
+f) With correct hash sum `ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a` run the `set_swupgrade_repo` command with incorrect hash-prefix:
+
+```bash
+
+provisioner set_swupgrade_repo 2.0.1 --source=/root/single_cortx.iso --hash="sha512:ff01da01d4304729bfbad3aeca53b705c1d1d2132e94e4303c1ea210288de12a"
+
+```
+
+Output should be the same as in c) sub-step.
+
+
+## EOS-18743: RELEASE.INFO file exists and has the necessary scheme
+
+The positive test scenario is the same as in [EOS-16883](#eos-16883-sw-upgrade-structure-validation)
+
+For negative test scenario need to create the wrong SW upgrade ISO bundle.
+Commands for manual ISO creation are listed in - [EOS-15766](#eos-15766-test-resume-update-process-after-reboot): Step-2: b)
+
+The better way to re-use ISO bundle created by the following command
+```bash
+pytest test/build_helpers/build_bundles
+```
+
+For example, it is enough to delete line which starts with `VERSION:` in file `cortx_iso/RELEASE.INFO`.
+And create the new ISO file with this change.
+
+The following command with broken `single_cortx.iso` should be failed
+```bash
+provisioner set_swupgrade_repo 2.0.1 --source=/root/single_cortx.iso
```

Comment:
The iso path would be: `/opt/isos`
Try not to have anything under `/root`

Reply:
It is only for testing purposes, I provided this path as just an example

Reply:
I changed path to `/opt/iso/single_cortx.iso`.

Reply:
> It is only for testing purposes, I provided this path as just an example

Agreed, but it's a good practice generally to avoid `/root`.